foo.js

```diff
@@ -0,0 +1,5 @@
+let x = 5;      // Declare x, give it the value of 5
+let y = x + 2;  // Declare y, give it the value of x + 2 

```

Comment:
```suggestion
const y = x + 2; // Declare y as a constant with the value of x + 2
```
_Generated using [OpenSauced](https://opensauced.ai/)._

Reply:
At this point I think it's just my browser. We can move forward since it clearly works and I can debug further on my end.

Reply:
One last thing I can recommend to debug is probably enabling the extension in incognito mode and trying it there. (You'll have to login and everything).